maven-surefire-common/src/test/java/org/apache/maven/plugin/surefire/util/DependenciesScannerTest.java

```diff
@@ -19,12 +19,17 @@
  * under the License.
  */
 
-import junit.framework.TestCase;
+import static org.junit.Assert.*;
```

Comment:
Not sure if checkstyle will break the build because of using star `*` in this import.